### PR TITLE
object-image-overlay: add swipe gestures [web-2114]

### PR DESCRIPTION
**Pull request by methodog**

### CODE REVIEW

File: src/components/blocks/object-image-overlay/_object-image-overlay.js [hunk 1]
```diff
@@ -304,6 +304,37 @@ const oicInit = () => {
             oic.advance(true);
           }
         };
+
+        oic.ontouchstart = (e4) => {
```
Comment (nickbergquist) [inline src/components/blocks/object-image-overlay/_object-image-overlay.js:304]: Possibly consider updating older DOM Level 0 event handling to DOM Level 2 standardised method, i.e.,

```
oic.addEventListener('touchstart', (e4) => {
  const startXY = [e4.touches[0].pageX, e4.touches[0].pageY];
  ...
});
```

File: src/components/blocks/object-image-overlay/_object-image-overlay.js [hunk 2]
```diff
@@ -304,6 +300,37 @@ const oicInit = () => {
...
+          const startXY = [e4.touches[0].pageX, e4.touches[0].pageY];
+          oic.ontouchmove = (e5) => {
+            const deltaXY = [e5.touches[0].pageX - startXY[0], e5.touches[0].pageY - startXY[1]];
+            if (Math.abs(deltaXY[0]) > Math.abs(deltaXY[1])
+              && (
+                (deltaXY[0] < 0 && oic._index < oicSeeds.length - 1)
+                || (deltaXY[0] > 0 && oic._index > 0)
+              )) {
+              if (Math.abs(deltaXY[0]) < 74) {
```
Comment (nickbergquist) [inline src/components/blocks/object-image-overlay/_object-image-overlay.js:313]: Magic number here with no indication of how it is derived.

File: src/components/blocks/object-image-overlay/_object-image-overlay.js (hunk 1 above)
Comment (methodog) [reply src/components/blocks/object-image-overlay/_object-image-overlay.js:304]: normally yes, but in this case it's to supersede any/all other touch event handlers

File: src/components/blocks/object-image-overlay/_object-image-overlay.js (hunk 2 above)
Comment (methodog) [reply src/components/blocks/object-image-overlay/_object-image-overlay.js:313]: yes, could do with commenting generally. that's just 74px, a minimum touchmove distance to activate a full swipe.

**Pull request closed without merging**